Sources/TSCUtility/Platform.swift

```diff
@@ -33,6 +34,10 @@ public enum Platform {
             if localFileSystem.isFile(AbsolutePath("/etc/debian_version")) {
                 return .linux(.debian)
             }
+            if localFileSystem.isFile(AbsolutePath("/system/bin/toolbox")) ||
+               localFileSystem.isFile(AbsolutePath("/system/bin/toybox")) {
+                return .android
+            }
```

Comment:
This function doesn't seem to be called anywhere, can remove if not needed.

Reply:
yah let's remove it if it's dead code.

Reply:
I was wrong, the variable it's used to initialize, `currentPlatform`, is used [in exactly one place](https://github.com/apple/swift-package-manager/blob/8b6eebd3d83fb0dd6e26ffba195ba5bcb143c356/Sources/PackageLoading/Target%2BPkgConfig.swift#L111). I suppose this might be useful for TSCUtility and llbuild someday too, [for which I've created a Termux package](https://github.com/termux/termux-packages/tree/master/packages/llbuild), so I'd like to keep this Android support.

Reply:
Any reason you chose these particular files vs `/system/build.prop`?

Reply:
No other reason that I'm familiar with toolbox/toybox, as they're sometimes invoked from the Termux command-line, and not build.prop, which is unreadable without root on my device. If there's a better file that signifies Android, glad to use that instead, just let me know.

Reply:
OK, that's fine for now, we can always update the check later if we need to.

Thanks for your contribution!